packages/sdks/src/functions/track.ts

```diff
@@ -38,12 +44,19 @@ const getTrackingEventData = async ({
   };
 };
 
-type EventProperties = {
-  type: Event['type'];
-  orgId: Event['data']['ownerId'];
-  contentId: Event['data']['contentId'];
-  [index: string]: any;
-};
+type EventProperties = Pick<Event, 'type'> &
+  Pick<Event['data'], 'ownerId' | 'contentId'> & {
+    /**
+     * This is the ID of the space that the content belongs to.
+     */
+    orgId: Event['data']['ownerId'];
```

Comment:
this should always be called `apiKey` externally and referenced as such in the jsDocs (e.g.the jsDoc should say "this is your public API key")